datasets/woz_dialogue/woz_dialogue.py

```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""WozDialogue: Reading Comprehension with Multiple Hops"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+_CITATION = """\
+@misc{wen2017networkbased,
+      title={A Network-based End-to-End Trainable Task-oriented Dialogue System},
+      author={Tsung-Hsien Wen and David Vandyke and Nikola Mrksic and Milica Gasic and Lina M. Rojas-Barahona and Pei-Hao Su and Stefan Ultes and Steve Young},
+      year={2017},
+      eprint={1604.04562},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DESCRIPTION = """\
+Wizard-of-Oz (WOZ) is a dataset for training task-oriented dialogue systems. The dataset is designed around the \
+task of finding a restaurant in the Cambridge, UK area. There are three informable slots (food, pricerange,area) \
+that users can use to constrain the search and six requestable slots (address, phone, postcode plus the three informable slots) \
+that the user can ask a value for once a restaurant has been offered.
+"""
+
+_HOMEPAGE = "https://github.com/nmrksic/neural-belief-tracker/tree/master/data/woz"
+
+_BASE_URL = "https://raw.githubusercontent.com/nmrksic/neural-belief-tracker/master/data/woz"
+
+
+class WozDialogue(datasets.GeneratorBasedBuilder):
+    """WozDialogue: Reading Comprehension with Multiple Hops"""
+
+    VERSION = datasets.Version("1.0.0")
+    BUILDER_CONFIGS = [
```

Comment:
If I understand correctly the configuration `it_en` has the dialogs in italian but the labels in english ?
Could you add a comment about this ?